### PR TITLE
adding a setchon in the read me for gentoo for dependens.

### PR DESCRIPTION
**Pull request by Codespace0x25**
i added the command requred to install all the depindins for runtime and build. excluding dmenu

### CODE REVIEW

**Comment by Codespace0x25** [comment]
If you see any misspellings let me know I'll fix it

**Comment by Codespace0x25** [comment]
im missing a depindinc 
the assimp it is media-lib/assimp
requied by the player.

File: readme.md [hunk 1]
```diff
@@ -126,6 +126,12 @@ If you are on Arch and would like to help with a PR that I can get merged into m
 ```bash
 sudo pacman -S xdotool dmenu xorg-server xorg-xwininfo xorg-xrandr protobuf base-devel zeromq libx11 libxcomposite libxtst libxext libxfixes spdlog fmt glfw-x11 mesa assimp sqlite
 ```
+#### Gentoo
+```bash
+ sudo emerge net-libs/zeromq x11-libs/libX11 x11-libs/libXcomposite x11-libs/libXtst x11-libs/libXext x11-libs/libXfixes dev-libs/protobuf dev-libs/spdlog dev-libs/libfmt glfw x11-libs/libGLw  dev-db/sqlite x11-misc/xdotool  protobuf dev-libs/pthreadpool media-libs/libass media-lib/assimp
+```
+> [!NOTE]
+> you may have some isses with use flags and or masked packages. you will have to figer that out on your own system.
```
Comment (Codespace0x25) [inline readme.md:134]: I'll change this to say 
" you may have some issues with use-flags and or masked packages, you will have to figure it out on your system as your use-flag configuration may differ from others systems."

**Comment by Codespace0x25** [comment]
also in order to make a .ebuild of for this program we will need to set up a release as to be able to download the src.

**Comment by collinalexbell** [comment]
Why excluding dmenu? It is a fundamental part of HackMatrix.

File: readme.md (hunk 1 above)
Comment (collinalexbell) [reply readme.md:134]: replace "figer" with "figure" 

"s/figer/figure" is how I normally comment about a substitution

**Comment by Codespace0x25** [comment]
Fair point I'll update that

File: readme.md (hunk 1 above)
Comment (Codespace0x25) [reply readme.md:134]: Ok I'll fix it

**Comment by Codespace0x25** [comment]
Ok I added dmenu in to the read me to install along with everything thang else

**Comment by collinalexbell** [comment]
❤️

**Pull request merged**